mindsdb_native/libs/backends/lightwood.py

```diff
@@ -86,7 +86,12 @@ def _ts_reshape(self, original_df):
         for i in range(len(df_arr)):
             for order_col in ob_arr + self.transaction.lmd['tss']['historical_columns']:
                 for ii in range(len(df_arr[i])):
-                    df_arr[i][order_col].iloc[ii] = [df_arr[i][order_col].iloc[ii]]
+                    try:
+                        df_arr[i][order_col].iloc[ii] = [df_arr[i][order_col].iloc[ii]]
+                    except:
```

Comment:
`except Exception`

Reply:
Can be more than one Exception type, I'll add the generic case